BE/OwnershipAndControl/CorporateControl.rdf

```diff
@@ -135,59 +134,65 @@
+		<skos:definition>party that shares capital, technology, human resources, risks, and benefits of an entity under shared control</skos:definition>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-be-oac-cctl;NonWhollyOwnedSubsidiary">
 		<rdfs:subClassOf rdf:resource="&fibo-be-oac-cctl;Subsidiary"/>
 		<rdfs:label>non-wholly owned subsidiary</rdfs:label>
-		<skos:definition>a subsidiary which is not a wholly owned subsidiary</skos:definition>
+		<skos:definition>subsidiary that is not 100 percent owned by its parent</skos:definition>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-be-oac-cctl;OverFiftyPercentControllingInterestCompany">
-		<rdfs:subClassOf rdf:resource="&fibo-be-oac-cctl;VotingShareholdingCompany"/>
-		<rdfs:label>over fifty percent controlling interest company</rdfs:label>
-		<skos:definition>a voting shareholding company that owns more than fifty (50) percent of the voting shares in another company</skos:definition>
+	<owl:Class rdf:about="&fibo-be-oac-cctl;OverFiftyPercentControllingInterestParty">
+		<rdfs:subClassOf rdf:resource="&fibo-be-oac-cctl;VotingShareholder"/>
+		<rdfs:label>over fifty percent controlling interest party</rdfs:label>
+		<skos:definition>voting shareholder that owns more than fifty (50) percent of the voting shares in some legal entity</skos:definition>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-be-oac-cctl;SignificantShareholdingCompany">
+	<owl:Class rdf:about="&fibo-be-oac-cctl;SignificantShareholder">
 		<rdfs:subClassOf rdf:resource="&fibo-be-oac-cpty;SignificantControllingInterestParty"/>
-		<rdfs:subClassOf rdf:resource="&fibo-be-oac-cctl;VotingShareholdingCompany"/>
-		<rdfs:label>significant shareholding company</rdfs:label>
-		<skos:definition>a company that owns a significant voting stake in another company</skos:definition>
+		<rdfs:subClassOf rdf:resource="&fibo-be-oac-cctl;VotingShareholder"/>
+		<rdfs:label>significant shareholder</rdfs:label>
+		<skos:definition>person or legal entity that owns a significant voting stake in another company</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote>Note that the concept of significance varies depending on the jurisdiction, and particularly with respect to reporting requirements.  For example, in some cases, three (3) percent ownership of any class or series of shares is considered significant, and in others it means more than five or ten percent of the total combined voting power across all classes of securities.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-be-oac-cctl;Subsidiary">
 		<rdfs:subClassOf rdf:resource="&fibo-be-oac-cctl;ControlledCompany"/>
 		<rdfs:label>subsidiary</rdfs:label>
-		<skos:definition>a company that is entirely or partially owned and entirely or partially controlled by another company</skos:definition>
+		<skos:definition>legal entity that is entirely or partially owned and entirely or partially controlled by another legal entity</skos:definition>
```

Comment:
We should use "genus / differentia" structure for definitions as described In
https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md#definitions

Reply:
I would agree in most cases, but this particular term is one where a number of users on our calls have been confused, so that played into the change as well.  All of these ownership and control ontologies need further work with respect to properties and definitions, especially in light of BE-44, which clarifies the top level lattice - I want to work through another pass with David and Dean, who were leading the BE effort for several years and raised some of these issues, and consider merging some of the ontologies in the process.

Reply:
OK, it makes sense. I'll accept this PR. Please make sure that definitions in BE follow "genus / differentia" structure.